### DISCUSSION TITLE
GitHub Copilot The new version no longer works

### DISCUSSION THREAD

**Discussion by machine-arch** [community/community#31206] [Copilot Conversations] [answered]
Hi, when my copilot updated automatically it stopped working, version 1.38 and older ones work fine but newer versions runtime status is 'Not yet activated',

 I also have Copilot installed on my home computer, and all versions work fine there.

on both computers vscode version is 1.70.2.

I tried uninstalling all extensions leaving only copilot but it didn't work, I also tried uninstalling copilot and reinstalling but that didn't work either, I also deleted vscode, deleted all its folders from the system then installed just copilot but that didn't work either.

Does anyone have any idea what is going on?

I connect to the work computer with VPN

**Comment by davecheney** [comment, marked as answer]
@machine-arch i'm sorry you are having difficulties. If your internet service provider, employer, or government is filtering your SSL traffic you may have difficulty using Copilot. Please see https://github.com/community/community/discussions/30716, which may help.

**Reply by machine-arch** [reply]
That I was using without problems before version 1.39?

**Reply by davecheney** [reply]
Please see https://github.com/orgs/community/discussions/30716

**Reply by machine-arch** [reply]
Thanks for the help. I found the problem and if anyone else has a similar problem, it is most likely that your network uses a proxy and you have to manually write the proxy configuration host and port in the editor.
